tests/lib/test_app.ml

```diff
@@ -87,3 +88,7 @@ module C = struct
 end
 
 include Functoria_app.Make(C)
+
+let register ?packages ?keys name jobs =
+  let init = [Functoria_app.(keys sys_argv)] in
+  register ?keys ?packages ~init name jobs
```

Comment:
Maybe it would make sense to add `Functoria_app.key` as the default `init` device.